src/Aquifer.Data/Entities/BibleBookContentEntity.cs

```diff
@@ -22,4 +27,38 @@ public class BibleBookContentEntity : IHasUpdatedTimestamp
 
     [SqlDefaultValue("getutcdate()")]
     public DateTime Updated { get; set; } = DateTime.UtcNow;
+
+    /// <summary>
+    /// POCO for deserializing the AudioUrls column.
+    /// </summary>
+    public sealed class AudioUrlsData
+    {
+        public required IReadOnlyList<Chapter> Chapters { get; set; }
```

Comment:
I wouldn't put `required` on these if they're being instantiated from a deserializer, because if it's not there it's gonna blow up. For example, you've got `Webm` and `Mp3` below as nullable. If they're actually null then it's fine, but if they're simply not there, the `required` is gonna cause the deserialize to blow up.

Reply:
TIL: https://learn.microsoft.com/en-us/dotnet/standard/serialization/system-text-json/required-properties.  I can remove `required` but at what point should I assume a data schema?  For example, I'd expect chapter number and verse number to still be required.

Reply:
[Fix commit](https://github.com/BiblioNexusStudio/aquifer-server/pull/443/commits/c5958ec535fecf230ab2929f1aba1eb78a0214d2) that assumes nested JSON objects/collections might be missing but if those objects/collections are present then they will have expected `string`/`number` properties. I could also go on the defensive for those if desired but I'd argue we should enforce better data at that point.